### ISSUE TITLE
Camera is not recognized after reboot

### ISSUE THREAD

**Issue by FunPsycho** [ccrisan/motioneyeos#1935]
### Preliminary Docs

I confirm that I have read the [CONTRIBUTING](https://github.com/ccrisan/motioneyeos/blob/master/.github/CONTRIBUTING.md) guide before opening this issue.

I confirm that I have read the [FAQ](https://github.com/ccrisan/motioneyeos/wiki/FAQ) before opening this issue.

### motionEyeOS Version

I am running motionEyeOS version: 0.40rc5 (server/Home Assistant Raspberry Pi 3 B+)  and 0.40 client camera (Raspberry Pi Zero W)

### Board Model

I am using the following board/model: (insert your board model here, e.g. Raspberry PI 3B+).
Pi 3 B+ for server
Pi 0 w for client

### Camera

I am using the following type of camera: Raspberry Pi o w with v2 camera 

### Network Connection

My motionEyeOS unit is connected to the network via: Wifi

### Log Files

I consider the following log files relevant to this issue:
Home Assistant Log
[Server Log.txt](https://github.com/ccrisan/motioneyeos/files/3244008/Server.Log.txt)

Camera Log
[boot.log](https://github.com/ccrisan/motioneyeos/files/3244009/boot.log)
[motioneye.log](https://github.com/ccrisan/motioneyeos/files/3244010/motioneye.log)
[motion.log](https://github.com/ccrisan/motioneyeos/files/3244011/motion.log)
[messages.log](https://github.com/ccrisan/motioneyeos/files/3244012/messages.log)

So my problem is that I setup the Add-On in Home assistant, then start up my RP0w with the MotionEyeOS image found here https://github.com/ccrisan/motioneyeos/releases/download/20190427/motioneyeos-raspberrypi-20190427.img.xz and after I boot that one up and connect to the browser the camera runs fine.  I can find it on the Server (Home Assistnat) and everything is great.  Then I change some settings on the Zero to make the image better and reboot.  After the reboot the Home Assistant Add on will not find the camera anymore, the web UI for the client camera still works.  

![image](https://user-images.githubusercontent.com/11483359/58751545-846fe600-8465-11e9-9d2f-b15cbc05bdab.png)

**Comment by FunPsycho** [comment]
So I rebinstalled MotionEyeOS onto my client RP0 and noticed that when it does connect to the Home Assistant Add on it is a different kind of camera, but when I change my settings and reset it the camera type is now a Simple MJPEG Camera.  

So I figured I'd just add that, got the streaming IP from the client and entered it into the server and now it adds the camera but there is no image. 

I see this error in the log

Cannot open device /dev/video0, exiting.

I restarted both and I will reinstall on Client when I get back home without changing any options and I think we can nail where this is happening.
